src/SprykerSdk/Spryk/Model/Spryk/Filter/DashToCamelCaseFilter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Filter;
+
+use Zend\Filter\FilterChain;
+use Zend\Filter\Word\DashToCamelCase;
+
+class DashToCamelCaseFilter implements FilterInterface
+{
+    protected const FILTER_NAME = 'dashToCamelCase';
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::FILTER_NAME;
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return string
+     */
+    public function filter(string $value): string
+    {
+        $filterChain = new FilterChain();
```

Comment:
should be is factory and with adaptor

Reply:
I asked René, he said that it's ok to have it like this.